vuln/npm/451.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": 451,
+  "title": "Cross-site Scripting (XSS) - Generic",
+  "overview": "[statics-server] XSS via injected iframe in file name when statics-server displays directory index in the browser",
```

Comment:
the `[statics-server]` seems a bit redundant to me.

Reply:
it takes it straight out of hackerone's title and that's what we've been working with for a while. I agree we should probably have a better overview format in general.